commcare-cloud-bootstrap/commcare_cloud_bootstrap.py

```diff
@@ -1,22 +1,18 @@
-from __future__ import absolute_import, print_function, unicode_literals
-
 import argparse
 import json
 import os
 import random
 import re
+import secrets  # New in 3.6
```

Comment:
Because I'm looking at the PR description I know this means "Python 3.6" but that may not be obvious in other contexts.
```suggestion
import secrets  # New in Python 3.6
```

Reply:
Yeah. After I wrote the PR description I was wondering whether we really need this comment at all. :shrug: Keeping it.